RecoTracker/MeasurementDet/plugins/TkGluedMeasurementDet.cc

```diff
@@ -139,7 +152,8 @@ bool TkGluedMeasurementDet::measurements( const TrajectoryStateOnSurface& stateO
 				      (theMonoDet->hasAllGoodChannels() || 
 				       testStrips(stateOnThisDet,gluedPlane,*theMonoDet)
 				       )
-				      ) /*Mono OK*/ || 
+				      ) /*Mono OK*/ 
+                                     && // was || 
```

Comment:
Technically speaking this is a "bug fix" unrelated to HIP mitigation in its current incarnation. Its effects can be mostly seen in runs with many "masked strips" such as 275828